### PR TITLE
ls: update Hindi translation

### PR DESCRIPTION
**Pull request by pranavupadhyay123**
can you please tag this PR for "hacktoberfest"

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — CHANGES_REQUESTED** [review]
Welcome to tldr and thanks for your contribution.

This page doesn't seem to align with an English page as we have `ls` in the [common directory](https://github.com/tldr-pages/tldr/blob/main/pages/common/ls.md) with commands that is the same between multiple platforms. Please translate that page instead.

Edit. This PR is already eligible for hacktoberfest as we have tagged our repos with the hacktoberfest label, once this PR gets merged your contribution will get counted there after a 7 day wait period.

**Review by kbdharun — APPROVED** [review]

**Comment by pranavupadhyay123** [comment]
hey, can you please label this PR for Hacktober Fest
`hacktoberfest-accepted`

**Comment by kbdharun** [comment]
> hey, can you please label this PR for Hacktober Fest
> `hacktoberfest-accepted`

Hi, we automatically label any PRs that weren't merged with this label after October 29th.

Regarding this PR, it will automatically get counted after another maintainer approves and merges it.

**Review by blueskyson — APPROVED** [review]

**Pull request merged**
